### DISCUSSION TITLE
Is it possible to bring back the 8v8 simul & tag feature in the latest IKEMEN GO Version?

### DISCUSSION THREAD

**Discussion by ghost** [ikemen-engine/Ikemen-GO#1421] [Q&A] [answered]
My honest question. because i'm curious if it works.

**Comment by Orange-Dolphin** [comment, marked as answer]
I thought this idea sounded funny so I decided to try and get it to work, unfortunately you do need to modify the source code, but luckily it's only 1 line.

In system.go, line 25, it says MaxSimul = 4, this controls how many characters can be on a team, and if this is not changed, it will crash with "Invalid Team Size" when trying to use them. If you change that line to MaxSimul = 8, then build the program, this crash won't happen.

You will also have to manually change the config.json to set each team size to 8, like NumTag 2, 8 instead of 2, 4

And with that, I was able to create 8v8 battles. 

I will note, it may be because my screenpack isn't set up for 8 players on a team, but when I'm playing as the fifth through eighth character on the team in Tag, the lifebars will not update. For me personally this is fine because I just wanted to see the chaos of 8 v 8, so I'm not going to look into it too much.

**Reply by ghost** [reply]
Ok, but how can i Modify the source code? (Sorry if i was too late.)

**Comment by ZyonZZ** [comment]
interesting...
but in this concept it seems that the engine is still very basic...

I would like it to be possible online...
that ikemen had its launcher..
and when you open it and see the IKEMENS creating online games...
in the same launcher that the rooms are seen.
You connect... and download the data from the ikemen HOST that your opponent has...
to start the game.
automate things a little.


also within the game... or from the same launcher if you like...
You had the option of 3 vs 3 games... and 6 players each choosing a character...
4 players... of 2 vs 2 teams

with an interface of controls moving, and grouping for one side of the team... (Soccer game style that many players can play and choose sides)
and online... but I think this is missing too much...

or also in Turns arcade or tag mode... 3 or 4 players... online
and that everyone can select a character... and during combat make player changes in TAG battles

If you have programming knowledge or know someone who can do it... and can help with mods, the community would appreciate it.
